src/EventStore.ClientAPI/Internal/ClusterDnsSeed.cs

```diff
@@ -0,0 +1,21 @@
+namespace EventStore.ClientAPI.Internal {
+	public class ClusterDnsSeed : IGossipSeed {
+		private readonly string _clusterDns;
+		private readonly bool _secure;
+		
+		public ClusterDnsSeed(string clusterDns, bool secure) {
+			_clusterDns = clusterDns;
+			_secure = secure;
+		}
+
+		public string ToHttpUrl() {
+			var scheme = _secure ? "https" : "http";
+
+			return string.Format("{0}://{1}/gossip?format=json", scheme, _clusterDns);
+		}
+
+		public string GetHostHeader() {
+			return "";
```

Comment:
Always returning `""` for the host header could be a problem when connecting to version 20.6 or cloud (for TLS connections). I will test it out to confirm. We may need to introduce an `EnableVersion20Compability` flag here to control whether the host header is sent or not.

Reply:
I originally tested on Cloud with no problem, which was running 20.6.1 at that time.

Reply:
I can confirm that it results in a certificate validation error with cloud